scipy/optimize/slsqp.py

```diff
@@ -159,19 +159,19 @@ def fmin_slsqp(func, x0, eqcons=(), f_eqcons=None, ieqcons=(), f_ieqcons=None,
 
     Notes
     -----
-    Exit modes are defined as follows ::
-
-        -1 : Gradient evaluation required (g & a)
-         0 : Optimization terminated successfully
-         1 : Function evaluation required (f & c)
-         2 : More equality constraints than independent variables
-         3 : More than 3*n iterations in LSQ subproblem
-         4 : Inequality constraints incompatible
-         5 : Singular matrix E in LSQ subproblem
-         6 : Singular matrix C in LSQ subproblem
-         7 : Rank-deficient equality constraint subproblem HFTI
-         8 : Positive directional derivative for linesearch
-         9 : Iteration limit reached
+    Exit modes are defined as follows :
+
+    -1 : Gradient evaluation required (g & a)
```

Comment:
these aren't being displayed correctly

Reply:
https://22669-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.optimize.fmin_slsqp.html#scipy.optimize.fmin_slsqp

Reply:
Thanks for catching that. I made them all on separate lines.

Reply:
It would be nice if the blank lines added here weren't necessary, but it might be an acceptable compromise.

Reply:
This is still not rendering properly: https://22691-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.optimize.fmin_slsqp.html#scipy.optimize.fmin_slsqp

I suspect you need an additional space before `-1`, to keep the left-most characters in each line aligned.

Reply:
Fixed!